### ISSUE TITLE
Python dependency versioning

### ISSUE THREAD

**Issue by chasemc** [mwang87/GNPS_LibraryDashboard#2]
Might want to think about adding strict dependency versions to the Dockerfile?
e.g. `RUN conda install -c anaconda pandas=1.0.3`

Fast way to specify all dependencies + versions:
https://github.com/jazzband/pip-tools

**Comment by mwang87** [comment]
Yeah great suggestion! I'm super lazy on first iterations. It has bitten me in the butt not being specific on versions so I should actually learn my lesson one of these days!
